xil_carma_cloud/SubsystemControllerParams.yaml

```diff
@@ -21,7 +21,10 @@ hardware_interface:
       excluded_namespace_nodes:
```

Comment:
Just a general comment for the PR - Could we add a note on whether this has been tested? Also could "carma-platform-4.5.0" be replaced with "CARMA Platform tag carma-system-4.5.0" (or similar)?